File: Backend-POS/POS.Main/POS.Main.Business.Payment/Models/Payment/SlipUploadResultModel.cs
namespace POS.Main.Business.Payment.Models.Payment;

public class SlipUploadResultModel
{
    public int OrderBillId { get; set; }
    public int SlipImageFileId { get; set; }
    public decimal? OcrAmount { get; set; }
    public string VerificationStatus { get; set; } = string.Empty;
    public decimal BillGrandTotal { get; set; }

    // ผลตรวจวันที่โอน
    public DateTime? OcrTransferDate { get; set; }
    public bool? IsDateToday { get; set; }

    // ผลตรวจเลขบัญชีปลายทาง
    public string? OcrAccountNumber { get; set; }
    public string? ShopAccountNumber { get; set; }
    public bool? IsAccountMatched { get; set; }
}

File: Frontend-POS/RBMS-POS-Client/src/app/core/api/models/slip-upload-result-model.ts
/* tslint:disable */
/* eslint-disable */
export interface SlipUploadResultModel {
  billGrandTotal?: number;
  isAccountMatched?: boolean | null;
  isDateToday?: boolean | null;
  ocrAccountNumber?: string | null;
  ocrAmount?: number | null;
  ocrTransferDate?: string | null;
  orderBillId?: number;
  shopAccountNumber?: string | null;
  slipImageFileId?: number;
  verificationStatus?: string | null;
}
